CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ option(TRACE_PARSER "Trace parser execution" OFF)
 option(TRACE_LEXER "Trace lexer execution" OFF)
 
 if(MSVC)
-  add_compile_options(/W1 /MP)
+  add_compile_options(/W1 /MP /wd5105)
```

Comment:
Is that flag the "ignore macro expansions producing `defined`" flag? If yes, a comment pointing out that it should be removed ASAP would be good.

Reply:
Yes, I'll add a comment for that.